test/api-navigation.test.js

```diff
@@ -18,6 +18,10 @@ describe('<api-navigation>', () => {
     return (await fixture(`<api-navigation summary selected="test1"></api-navigation>`));
   }
 
+  async function arrangedFixture() {
+    return (await fixture(`<api-navigation rearrangeEndpoints="true"></api-navigation>`));
```

Comment:
Technically HTML attributes are always considered lowercase. This is not a mistake but we should have common coding standards.